docs/core/allocation.qmd

```diff
@@ -14,13 +14,12 @@ within this *Allocation* section the main network is also considered to be a sub
 # The high level algorithm
 The allocation algorithm contains 3 types of optimization:
 
-- `internal_sources`, where flows are allocated within a subnetwork by only using sources inside the subnetwork;
```

Comment:
Maybe this documentation can remain unchanged, as it refers to the optimization types, not the names of the functions.